src/state/browserHistory/index.ts

```diff
@@ -11,26 +9,28 @@ export interface Site {
 }
 
 interface BrowserHistoryStore {
-  recent: Site[];
+  recents: Site[];
   addRecent: (site: Site) => void;
-  getRecent: () => Site[];
 }
 
 const MAX_RECENT_SIZE = 1000;
 
-export const browserHistoryStore = createStore<BrowserHistoryStore>((set, get) => ({
-  recent: [],
-  addRecent: (site: Site) => {
-    let newRecents = [site, ...get().recent];
-    if (newRecents.length > MAX_RECENT_SIZE) {
-      newRecents = newRecents.slice(0, MAX_RECENT_SIZE);
-    }
-
-    set({
-      recent: newRecents,
-    });
-  },
-  getRecent: () => get().recent,
-}));
+export const useBrowserHistoryStore = createRainbowStore<BrowserHistoryStore>(
+  set => ({
+    recents: [],
 
-export const useBrowserHistoryStore = create(browserHistoryStore);
+    addRecent: (site: Site) => {
+      set(state => {
+        let newRecents = [site, ...state.recents];
+        if (newRecents.length > MAX_RECENT_SIZE) {
+          newRecents = newRecents.slice(0, MAX_RECENT_SIZE);
+        }
+        return { recents: newRecents };
+      });
+    },
+  }),
+  {
+    storageKey: 'browserHistory',
+    version: 0,
```

Comment:
Should we bump this?

Reply:
I think because it's hooked up to a new MMKV instance within `createBrowserStore`, the old data won't exist there